tests/tools.vitruv.applications.pcmjava.tests/src/tools/vitruv/applications/pcmjava/tests/pcm2java/javahelper/JavaCreatorsUtil.xtend

```diff
@@ -0,0 +1,148 @@
+package tools.vitruv.applications.pcmjava.tests.pcm2java.javahelper
+
+import edu.kit.ipd.sdq.activextendannotations.Utility
+import org.emftext.language.java.classifiers.Class
+import org.emftext.language.java.classifiers.ClassifiersFactory
+import org.emftext.language.java.classifiers.Interface
+import org.emftext.language.java.containers.CompilationUnit
+import org.emftext.language.java.containers.ContainersFactory
+import org.emftext.language.java.containers.Package
+import org.emftext.language.java.generics.GenericsFactory
+import org.emftext.language.java.members.ClassMethod
+import org.emftext.language.java.members.Field
+import org.emftext.language.java.members.InterfaceMethod
+import org.emftext.language.java.members.MembersFactory
+import org.emftext.language.java.types.Boolean
+import org.emftext.language.java.types.Int
+import org.emftext.language.java.types.NamespaceClassifierReference
+import org.emftext.language.java.types.PrimitiveType
+import org.emftext.language.java.types.TypesFactory
+import org.emftext.language.java.types.Void
+import org.palladiosimulator.pcm.repository.PrimitiveTypeEnum
+
+import static extension edu.kit.ipd.sdq.commons.util.java.lang.IterableUtil.*
+
+@Utility
+class JavaCreatorsUtil {
```

Comment:
I did not check but would guess that this class is copied from the Java<->UML tests? In that case please add a TODO comment that the utility classes should be unified.